pype/modules/ftrack/lib/avalon_sync.py

```diff
@@ -1258,10 +1253,11 @@ def prepare_ftrack_ent_data(self):
                 tasks = {}
                 for task_type in task_types:
                     task_type_name = task_type["name"]
-                    task_type_def = task_type_mapping.get(task_type_name) or {}
-                    short_name = task_type_def.get("short_name")
+                    # Set short name to empty string
+                    # QUESTION Maybe better would be to lower and remove spaces
+                    #   from task type name.
                     tasks[task_type_name] = {
-                        "short_name": short_name or task_type_name
+                        "short_name": ""
```

Comment:
This change the logic. Short name of task type is always set to empty string. Maybe we should fill it all the time.

Reply:
But this only applies to any new task types being added to the project correct?